invenio_github/templates/semantic-ui/invenio_github/helpers.html

```diff
@@ -6,28 +6,27 @@
   Invenio is free software; you can redistribute it and/or modify it
   under the terms of the MIT License; see LICENSE file for more details.
 #}
-{%- macro doi_badge(doi, github_id, modal_name='') %}
+{%- macro doi_badge(doi, doi_url, github_id) %}
 {%- block doi_badge scoped %}
-  {% set doi_url = url_for('invenio_github_badge.latest_doi', github_id=github_id, _external=True) %}
-  {% set image_url = url_for('invenio_github_badge.index', github_id=github_id, _external=True) %}
-  {% set modal_name = modal_name or (doi | replace('/', '-')) %}
-  <span class="get-badge" data-toggle="tooltip" data-placement="bottom" title="Get the DOI badge!">
-    <img data-toggle="modal" data-target="[data-modal='{{ modal_name }}']" src="{{ image_url }}" alt="{{ doi }}"/>
-  </span>
-  <div class="modal fade badge-modal" data-modal='{{ modal_name }}'>
-    <div class="modal-dialog">
-      <div class="modal-content">
-        <div class="modal-body">
-          <h4>DOI Badge</h4>
-          <p>
-            <small class="text-muted">
-              This badge points to the latest released version of your repository. If you want a DOI badge for a specific release, please follow the DOI link for one of the specific releases and grab badge from the archived record.
-            </small>
-          </p>
-          {% from "invenio_formatter/macros/badges.html" import badges_formats_list %}
-          {{ badges_formats_list(image_url, doi_url)}}
-        </div>
-      </div>
+  {% set image_url = url_for('invenio_github_badge.index', repo_github_id=github_id, _external=True) %}
+  <a class="ui blue image label">
+    <img id="record-doi-badge" src="{{ url_for('invenio_formatter_badges.badge', title='DOI', value=doi, ext='svg', _external=True, _scheme='https') }}" alt="{{ doi }}">
+  </a>
+
+  <div id="doi-modal" class="ui modal fade">
```

Comment:
```suggestion
  <div class="ui modal fade doi-modal">
```
Same reason as above. You can use the class as selector in the js code.